format/format_test.go

```diff
@@ -336,6 +336,52 @@ a[_x[y]]`,
 			expected: `_x
 a[_x[y][[z, w]]]`,
 		},
+		{
```

Comment:
Do we have an example of how this would look prior to this fix ? I would help to understand this fix (at least to me 😅 )

Reply:
```
[
    _,
    "foo",
] = split( ... )
```

🙈